### DISCUSSION TITLE
post/terminal-ninja/

### DISCUSSION THREAD

**Discussion by giscus** [basnijholt/nijho.lt#8] [General]
# post/terminal-ninja/

A setup for maximal productivity and minimal keystrokes, with `zsh`, `oh-my-zsh`, `keychain`, `starship`, `autoenv`, `z`, `zsh-autosuggestions`, and more.

https://www.nijho.lt/post/terminal-ninja/

**Comment by basnijholt** [comment]
This is my first blogpost. Just testing whether commenting works :smile:
